src/rgw/rgw_process.cc

```diff
@@ -199,6 +199,25 @@ int process_request(RGWRados* store, RGWREST* rest, RGWRequest* req,
 
   req->log_format(s, "op status=%d", op_ret);
   req->log_format(s, "http status=%d", http_ret);
+  switch (http_ret / 100) {
+    case 1:
+      perfcounter->inc(l_rgw_http_1xx_count);
+      break;
+    case 2:
+      perfcounter->inc(l_rgw_http_2xx_count);
+      break;
+    case 3:
+      perfcounter->inc(l_rgw_http_3xx_count);
+      break;
+    case 4:
+      perfcounter->inc(l_rgw_http_4xx_count);
+      break;
+    case 5:
+      perfcounter->inc(l_rgw_http_5xx_count);
+      break;
+    default:
+      perfcounter->inc(l_rgw_http_NULL_count);
+  }
```

Comment:
this could be a helper function in rgw_rest.cc so it doesn't get duplicated in librgw.cc